modules/macos_utils/manifests/uninstall_homebrew.pp

```diff
@@ -0,0 +1,13 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+class macos_utils::uninstall_homebrew {
+
+    exec { 'run_homebrew_uninstaller':
+        path      => '/usr/bin:/usr/sbin:/bin',
+        command   => 'ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/uninstall)"',
+        onlyif    => 'test -e /usr/local/Homebrew',
```

Comment:
I adjusted like this for my testing:
https://github.com/mozilla-platform-ops/ronin_puppet/blob/f81ed37e0e2b9a6994eb751a6b1994973a9c6e24/modules/macos_utils/manifests/uninstall_homebrew.pp

I had to change to checking for the brew binary because the uninstaller leaves /usr/local/Homebrew (maybe just the new one does this).
maybe we need to delete the directory after the uninstall?

Reply:
The uninstall also checks for the brew bin so testing for that is the right option